index.js

```diff
@@ -54,6 +54,8 @@ LogWatcher.prototype.start = function () {
   var fileSize = fs.statSync(this.options.logFile).size;
   var players = [];
   var gameOverCount = 0;
+  var data = '';
```

Comment:
When reading ends with partial line (does not end in \n) the pop will but that part in data. Next time when rest of the line is read they are combined to get the complete line. initially its the empty string